seeds/initial.sql

```diff
@@ -0,0 +1,157 @@
+INSERT INTO shops (id, name, address, lat, long)
```

Comment:
If this gets cumbersome, you could always use a query builder to generate the seeds for you too. I imagine you typed this one by hand 😬